.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Intel Corporation
+name: Pull Request
+
+on: [pull_request]
+
+concurrency:
+  group: ${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+  
+
+jobs:
+  build:
+    env:
+      DOCKER_REGISTRY: "docker.io/"
+      DOCKER_REPOSITORY: "badhrinathpa/"
```

Comment:
Open item to check with ONF infra team ? @amolonf  would you be able to get this work with omec-project